src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Https/ClientCredentialTypeTests.cs

```diff
@@ -137,7 +137,6 @@ public static void DigestAuthentication_RoundTrips_Echo()
 
     [Fact]
     [OuterLoop]
-    [ActiveIssue(604, PlatformID.AnyUnix)]
     public static void NtlmAuthentication_RoundTrips_Echo()
```

Comment:
LGTM.
I ran this PR on Linux, and now that these tests are no longer excluded, they are run and they fail.
Should we open a new issue and keep an [ActiveIssue] on these or just leave them failing and investigate the failure?  Don't block the merge for this question.

    Https_ClientCredentialTypeTests.NtlmAuthentication_RoundTrips_Echo [FAIL]
        Test Case: NtlmAuthentication FAILED with the following errors:     Error: Unexpected exception was caught while doing the basic echo test for variation...
        'BasicHttpBinding with NTLM authentication'
        Exception: System.ServiceModel.CommunicationException: An error occurred while sending the request. ---> System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.Http.CurlException: Peer certificate cannot be authenticated with given CA certificates
           at System.Net.Http.CurlHandler.MultiAgent.FinishRequest(EasyRequest completedOperation, CURLcode messageResult)